src/scripts/main.js

```diff
@@ -1,7 +1,39 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const createNotificationElement = () => {
+    const notification = document.createElement('div');
+
+    notification.className = `notification ${type}`;
+    notification.style.position = 'fixed';
+    notification.style.top = `${posTop}px`;
+    notification.style.right = `${posRight}px`;
+
+    const titleElement = document.createElement('h2');
+
+    titleElement.className = 'title';
+    titleElement.textContent = title;
```

Comment:
Instead of creating and appending elements one by one, you could create a template string with all the HTML and use `innerHTML` to add it to the notification. This would make your code more readable and efficient.